api/src/score/ro/raw-data.ro.ts

```diff
@@ -48,7 +48,7 @@ export class RawDataRo {
         asks: item.asks,
         assignedTo: item?.assignedTo?.displayName || '-',
         lastUpdated: item.updatedAt.toString(),
-        status: item.status,
+        status: item.status?.name || '-',
```

Comment:
shouldn't the default be `RECEIVED` ?

Reply:
Default `RECEIVED` in database. I thought that if it cannot get status from BE, then there was something wrong and we need to display that rather than showing random status